client/src/app/auth/auth-form/auth-form.component.sass

```diff
@@ -0,0 +1,27 @@
+.container
+    display: flex
```

Comment:
MINOR add, please,   ;   in the end of lines

Reply:
SASS doesn't use semicolumns at the end of lines

Reply:
So i should switch all style file extensions to scss?

Reply:
no, let it stay like that